File: test/aidbox_sdk/snapshots/hl7-fhir-r4-core/Basic.cs
using Aidbox.FHIR.Base;
using Aidbox.FHIR.Utils;

namespace Aidbox.FHIR.Resource;

public class Basic : DomainResource, IResource
{
    public required Base.CodeableConcept Code { get; set; }
    public Base.ResourceReference? Author { get; set; }
    public string? Created { get; set; }
    public Base.ResourceReference? Subject { get; set; }
    public Base.Identifier[]? Identifier { get; set; }
}

File: test/aidbox_sdk/snapshots/python/hl7-fhir-r4-core/Basic.py
from typing import Optional, List
from base import CodeableConcept
from base import DomainResource
from base import Reference
from base import Identifier

class Basic(DomainResource):
    code: CodeableConcept
    author: Optional[Reference] = None
    created: Optional[str] = None
    subject: Optional[Reference] = None
    identifier: Optional[List[Identifier]] = None